src/test/java/org/opentripplanner/middleware/utils/ItineraryUtilsTest.java

```diff
@@ -165,6 +152,9 @@ void canCheckAllItinerariesExist(boolean insertInvalidDay, String message) throw
         // Days not monitored had an error response, so the check should return invalid for those days.
         assertFalse(existence.wednesday.isValid());
         assertFalse(existence.friday.isValid());
+
+        // Make sure all mpcks were used
```

Comment:
"mocks" and not "mpcks"

Reply:
Thanks, fixed in https://github.com/ibi-group/otp-middleware/pull/240/commits/aadf6e7194991a1d375187325b43f70b6004b780.